src/multiplayer-servers/getting-started/setup-third-party-oauth.md

```diff
@@ -0,0 +1,150 @@
+
+## Overview
+
+GameFabric supports Third-Party OAuth Integration with every OAuth-compatible Identiy Provider, allowing you to reuse your existing authentication flows, without giving GameFabric access to your sensitive authentication data.
+
+## Prerequisites
+
+Before setting up a Third-Party OIDC Provider, ensure the following conditions are met:
+
+- You have access to an active GameFabric installation with permissions to modify OAuth Providers or the default user. 
+- You have an OIDC ([OpenID Connect](https://openid.net/developers/how-connect-works/)) provider, including a **Client ID** and **Client Secret**.
+
+## Setting up a new OIDC Provider
+
+Navigate to the "Access Management" section from the Sidebar Menu, and then into the "OIDC Providers" page. You may add a new OIDC Provider via the **Add Provider** button.
+
+![Add Provider Screenshot](images/authentication/create-oidc-provider-main-screenshot.png)
+
+For outstanding providers such as Microsoft ENTRA or Google Identity Platform; following instructions in [Preliminary Configurations](#preliminary-configurations) are sufficient to complete the integration.
+
+If you are using a non-standardized OIDC Provider, or would like to further customize the authentication flow (modifying prompt behaviour of your OIDC Provider, etc.) please refer to [Advanced Configurations](#advanced-configurations).
+
+### Preliminary configurations
+
+On the "General" step, enter a desired **ID** and **Display Name** for your provider. The OIDC Provider you are about to add will be saved under this **ID** and **Display Name** on GameFabric.
+
+![Provider Details Screenshot](images/authentication/create-oidc-provider-firststep-screenshot.png)
+
+On the "Provider" step, you need to specify the **Issuer URL** of your OIDC provider. This URL points to the OIDC Discovery Document of that particular provider, and GameFabric uses this Discovery Document for integrating with that provider.
+
+GameFabric has to be configured as an external application in the OIDC Provider, for the provider to assign **Client ID & Secret** ([Client ID & Secret](https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/)) for GameFabric. GameFabric requires this to connect to your provider.
+
+Fill in the **Client ID & Secret** to their respective text inputs, and click "Next".
+
+![Issuer Configuration Screenshot](images/authentication/create-oidc-provider-secondstep-screenshot.png)
+
+For OIDC Providers with standard behaviour no further configuration is necessary and you may click "Create" button to finish adding your provider.
+
+![Basic Provider Create Screenshot](images/authentication/create-oidc-provider-basic-finish.png)
+
+### Advanced Configurations
+
+This section allows further configuration of **Scopes & Claims**, **Prompt**, **Claim Mapping** and **Provider Discovery Override**.
+
+#### Scopes & Claims
+
+GameFabric requests further data about the authenticated user using **Scopes & Claims**.
```

Comment:
`further` is not right here. Use `additional` instead in all following sentences.
```suggestion
GameFabric requests additional data about the authenticated user using **Scopes & Claims**.
```